include/staff/email.inc.php

```diff
@@ -324,9 +334,28 @@
                  &nbsp;
                  <label><input type="radio" name="smtp_auth"  value="0"
                      <?php echo !$info['smtp_auth']?'checked':''; ?> /> <?php echo __('No'); ?></label>
+                 &nbsp;
+                 <label><input type="checkbox" name="smtp_auth_creds" value="1"
+                     <?php echo $info['smtp_auth_creds']?'checked':''; ?> />Use Separate Authentication</label>
```

Comment:
Put a space before and make text translatable.

Reply:
Done